rest-server/src/test/resources/nubesgen/terraform/app-service-python/terraform/modules/app-service/main.tf

```diff
@@ -0,0 +1,65 @@
+terraform {
+  required_providers {
+    azurecaf = {
+      source  = "aztfmod/azurecaf"
+      version = "1.2.11"
+    }
+  }
+}
+
+resource "azurecaf_name" "app_service_plan" {
+  name          = var.application_name
+  resource_type = "azurerm_app_service_plan"
+  suffixes      = [var.environment]
+}
+
+# This creates the plan that the service use
+resource "azurerm_app_service_plan" "application" {
+  name                = azurecaf_name.app_service_plan.result
+  resource_group_name = var.resource_group
+  location            = var.location
+
+  kind     = "Linux"
+  reserved = true
+
+  tags = {
+    "environment"      = var.environment
+    "application-name" = var.application_name
+  }
+
+  sku {
+    tier = "Free"
+    size = "F1"
+  }
+}
+
+resource "azurecaf_name" "app_service" {
+  name          = var.application_name
+  resource_type = "azurerm_app_service"
+  suffixes      = [var.environment]
+}
+
+# This creates the service definition
+resource "azurerm_app_service" "application" {
+  name                = azurecaf_name.app_service.result
+  resource_group_name = var.resource_group
+  location            = var.location
+  app_service_plan_id = azurerm_app_service_plan.application.id
+  https_only          = true
+
+  tags = {
+    "environment"      = var.environment
+    "application-name" = var.application_name
+  }
+
+  site_config {
+    linux_fx_version          = "PYTHON|3.9"
+    always_on                 = false
+    use_32_bit_worker_process = true
```

Comment:
This should be false

Reply:
Are you sure? I had bugs on Azure Functions and they told me to always put this to true. Maybe we can discuss with them?

Reply:
Checked with them. This is a legacy thing for Windows apps.

Reply:
Thanks! I'll update this everywhere!